### ISSUE TITLE
Unable to login: SQRL Login Page Expires

### ISSUE THREAD

**Issue by ohryan** [kalaspuffar/wordpress-sqrl-login#46]
**Describe the bug**
When I attempt to log in with SQRL I consistently receive an error. I have not been able to successfully login with SQRL.

**To Reproduce**
1. Open an new incognito window
2. Navigate to /wp-login.php
3. Click "Log in with username and password "
4. Scan QR code with Android app.
5. Android app returns the error "The SQRL login apge has expired. Please refresh the page, then log in with SQRL."
6. Refresh the page.
7. Scan QR code with Android app.
8. Same error as step 5 above.

**Expected behavior**
A successful login with SQRL.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox 69.0.2

**Smartphone (please complete the following information):**
 - Device: Pixel 3
 - OS: Android 10

**Additional context**
- The WordPress setting "Anyone posts a comment " is enabled. 
- Apache produces the following log entries;
```
[Sat Oct 12 15:58:28.292456 2019] [php7:notice] [pid 596] [client 108.162.241.58:36704] Missing transient session
[Sat Oct 12 15:58:28.292765 2019] [php7:notice] [pid 596] [client 108.162.241.58:36704] Failed response: Array\n(\n    [0] => ver=1\n    [1] => tif=20\n    [2] => sin=0\n)\n
```

**Comment by kalaspuffar** [comment]
Hi @ohryan 

An important detail here is: Have you associated the identity with an account?

Without the setting "Anyone can register" you will get this response for all logins. If the user is associated with the profile page it should work without this setting.

If your account is associated you should see this on the bottom of your profile page.

![profilepage](https://user-images.githubusercontent.com/1424084/66713049-eb06f800-eda5-11e9-8c22-0905fe8bf1f8.png)

Hope this helps.

Best regards
Daniel

**Comment by ohryan** [comment]
No, I have not associated an account with my user. Attempting to do so produces the same error.

**Comment by kalaspuffar** [comment]
Hi @ohryan 

I was in the middle of a rewrite and updating the test suite, so I had a bit to fix. Have you seen this problem recently, or is this issue fixed for you?

Best regards
Daniel
